packages/webapp/src/components/AvatarCustom.tsx

```diff
@@ -20,3 +21,53 @@ export const AvatarCustom: React.FC<Props> = ({ displayName, ...props }) => {
         </Avatar>
     );
 };
+
+type OrgProps = {
+    email?: string | null;
+    displayName: string;
+} & AvatarProps;
+
+const knownEmailProviders = new Set<string>([
+    'aol.com',
+    'duck.com',
+    'example.com',
+    'free.fr',
+    'gmail.com',
+    'gmx.de',
+    'googlemail.com',
+    'hey.com',
+    'hotmail.co.uk',
+    'hotmail.com',
+    'hotmail.fr',
+    'icloud.com',
+    'live.com',
+    'mailinator.com',
+    'me.com',
+    'orange.fr',
+    'outlook.com',
+    'proton.me',
+    'protonmail.com',
+    'qq.com',
+    'yahoo.co.uk',
+    'yahoo.com'
+]);
```

Comment:
According to Claude we could add a few more:

> Additional Google domains (gmail variants)
> More Yahoo regional domains (.fr, .de, .co.jp)
> Popular European providers (web.de, mail.ru, yandex.ru)
> Major Asian providers (yeah.net)
> Additional Microsoft domains (outlook.de, outlook.fr, msn.com)
> Business-focused providers (zoho.com)
> Additional French providers (laposte.net, sfr.fr)
> German providers (gmx.net, gmx.com)
> More Apple domains (mac.com)

did you get this list from existing customer emails?

Reply:
I got the list from our user base and top providers but not exhaustive, I'll add some more